### PR TITLE
Ft/footer

### PR DESCRIPTION
**Pull request by AdelineA**

### CODE REVIEW

**Review by gasangw — APPROVED** [review]
Hello @AdelineA 🟢 
**STATUS APPROVED** ✔️ 

- Correct implementation of the requirements.

- [x] [**OPTIONAL**] Please always add the pull request description. This is useful because it can help other users or even team members to know what you did in a particular file or PR

File: ReadMe.md [hunk 1]
```diff
@@ -726,4 +726,8 @@ remote: Resolving deltas: 100% (6/6), completed with 3 local objects.
 To github.com:AdelineA/Gym-Bundle-Exercises-Solutions.git
    19da6c2..52ed76b  main -> main
 
-```
\ No newline at end of file
+```
+# Bundle 4
+## Exercise 2
+
```
Comment (gsngdavid) [inline ReadMe.md:732]: Consider adding terminal history below the exercise

File: home.html
Comment (gsngdavid) [inline home.html:1]: 👍

**Review by gsngdavid — APPROVED** [review]
Looks good

File: home.html
Comment (UmutesiMMA) [inline home.html:1]: 👍🏾

File: ReadMe.md (hunk 1 above)
Comment (UmutesiMMA) [reply ReadMe.md:732]: I'd like to remind you to add the terminal history

**Pull request merged**